design/gateway-apis-implementation.md

```diff
@@ -27,22 +27,22 @@ In terms of the APIs themselves, the Gateway API has 3 primary API resources (ta
 In Contour, we've previously solved a lot of the same problems with HTTPProxy (and IngressRoute before it).
 That functionality can be described in the Gateway API by HTTPRoutes and TLSRoutes, as they describe Layer 7 ingress, as Contour does today.
 
-Other types of routes include TCPRoutes and UDPRoutes, which are intended for Layer 4 load balancers. Implementations may also define their own Route objects, at the cost of interoperability.
+Other types of routes include TCPRoutes and UDPRoutes, which are intended for Layer 4 load balancers.
+Implementations may also define their own Route objects, at the cost of interoperability.
 
 In terms of its stated goals, Contour is aiming at being an ingress controller - that is, a Layer 7 proxy with some api gateway functions.
 Currently, Contour provides "TCP Proxying" that allows the forwarding of TLS streams based on the SNI information, which is precisely what the Gateway API TLSRoute object is for.
 If Project Contour (the organisation) does add support for TCP and UDP forwarding, it will not be in the `projectcontour/contour` repo, but will be a separate repo.
 
-This design is intended to cover the initial `v1alpha1` release of the Gateway API. We will aim to implement the core featureset of the APIs at that point.
+This design is intended to cover the initial alpha releases of the Gateway API. 
+We will aim to implement the core featureset of the APIs at that point.
 We will then work with the upstream community on features that Contour and HTTPProxy currently support, but the Gateway API do not, and how best to represent those features in the APIs.
-So, there are some features that are currently gaps for Contour (wildcard domain names, and exact path matching seem like big ones),
+So, there are some features that are currently gaps for Contour (wildcard domain names, and exact path matching, for example),
```

Comment:
nit: Aren't both of these supported now?

Reply:
yup!